x/clp/keeper/calculations.go

```diff
@@ -16,53 +17,27 @@ func SwapOne(from types.Asset,
 	sentAmount sdk.Uint,
 	to types.Asset,
 	pool types.Pool,
-	normalizationFactor sdk.Dec,
-	adjustExternalToken bool,
-	pmtpCurrentRunningRate sdk.Dec) (sdk.Uint, sdk.Uint, sdk.Uint, types.Pool, error) {
+	pmtpCurrentRunningRate sdk.Dec,
+	marginEnabled bool) (sdk.Uint, sdk.Uint, sdk.Uint, types.Pool, error) {
```

Comment:
@AustinoBombino this need to live within the `calculations_FEATURE_TOGGLE_MARGIN_CLI_ALPHA.go` file instead as you are changing the signature of the function that if remains in this file will be available for production code, so instead as you want to make it available only when the feature toggle Margin is enabled you could one of those two options:

Option A:
1) move the entire `SwapOne` function with the margin changes to `calculations_FEATURE_TOGGLE_MARGIN_CLI_ALPHA.go`
2) create a new `calculations_NO_FEATURE_TOGGLE_MARGIN_CLI_ALPHA.go` file that will contain the header:
```
//go:build !FEATURE_TOGGLE_MARGIN_CLI_ALPHA
// +build !FEATURE_TOGGLE_MARGIN_CLI_ALPHA
```
and the original `SwapOne` function

Option B:
1) You follow the steps in Option A
2) You create an inner function that will live within `calculations.go` that will contain the code logic that are common between those two versions of the SwapOne function so there are less code to maintain in cases changes occur in this part of the code.